### DISCUSSION TITLE
getting error with isPlainObject when using Slate in a Javascript project: TypeError: Object(...) is not a function

### DISCUSSION THREAD

**Discussion by mahuzedada** [ianstormtaylor/slate#4585] [Q&A]
I am using slate in a javascript project. When i load the page I get the following error: `TypeError: Object(...) is not a function`.
The error seems to be thrown in this function:

`
  isText(value) {
    return isPlainObject(value) && typeof value.text === 'string';
  },
`

Any idea what might be causing this?

**Comment by aleksandrenko** [comment]
Second that

**Comment by siyasiyasiya** [comment]
I am also getting this error, did anyone find a solution? @aleksandrenko @mahuzedada
